README.md

```diff
@@ -115,6 +115,10 @@ Please check the Vapor Demo source code [here](https://github.com/meilisearch/me
 
 Please check the Perfect Demo source code [here](https://github.com/meilisearch/meilisearch-swift/tree/master/Demos/PerfectDemo).
 
+## Testing
+
+To run the unit and instrumented tests, run the script  `./run_test.sh`.
+
```

Comment:
```suggestion
```

Reply:
There is already a [section about testing](https://github.com/meilisearch/meilisearch-swift/blob/master/CONTRIBUTING.md#tests-and-linter) in the contributing guidelines. Could you please modify this section instead?